sympy/solvers/solveset.py

```diff
@@ -886,9 +882,7 @@ def solveset(f, symbol=None, domain=S.Complexes):
 
     if f.is_Relational:
         if not domain.is_subset(S.Reals):
-            raise NotImplementedError("Inequalities in the complex domain are "
-                                      "not supported. Try the real domain by"
-                                      "setting domain=S.Reals")
+            return ConditionSet(Lambda(symbol, f), S.Complexes)
```

Comment:
I think this should not be returned unevaluated. Users should know they can't solve complex inequalities from `solveset`.